src/demo/demoUtils.ts

```diff
@@ -145,7 +145,11 @@ export const getInitSdkOptions = (): SdkOptions => {
       ? SAMPLE_LOCALE
       : queryParamToValueString.language
 
-  const steps: Array<StepConfig> = [{ type: 'welcome' }]
+  const steps: Array<StepConfig> = [
+    { type: 'welcome' },
+    { type: 'userConsent' },
+    { type: 'auth' },
```

Comment:
This step should only be requested if it's we are testing auth, otherwise there will be a log in the console complaining about the fact that auth is not a step.